examples/run_tests.py

```diff
@@ -41,9 +41,9 @@
 # edit this dictionary to add or remove examples.
 # keys are the examples folder, and values are the configuration files
 examples = (
-    ("docking-protein-DNA"         , "docking-protein-DNA-test.cfg"),
-    ("docking-protein-DNA"         , "docking-protein-DNA-mdref-test.cfg"),
-    ("docking-protein-homotrimer"  , "docking-protein-homotrimer-test.cfg"),
+    ("docking-protein-DNA", "docking-protein-DNA-test.cfg"),
```

Comment:
This file is outside the `tox.ini` file. I added those spaces on purpose, don't remove them :wink:

Reply:
Please indicate the stuff that you added for vanity so I know what I can change or not 😉

Reply:
every PR I do and merge to the `main` is done on purpose. Nothing is left unnoticed :sunglasses: tox controlled lint files are defined here -> https://github.com/haddocking/haddock3/blob/593fa7d3f63a09d826056ed4b8e54e1955d0a35c/tox.ini#L66

Reply:
sure, made another PR adding the documentation not to remove them, are these also present on somewhere else? if so add them in https://github.com/haddocking/haddock3/pull/299